### PR TITLE
[feat] Creation of Home Page , basic routing & context

### PR DESCRIPTION
**Pull request by Radheshyam1234**
1.Done basic react project setup 
2. Created homepage component
3. Done basic routing using React router
4. Added state context for wishlist, cart and ordered items.

### CODE REVIEW

File: front-kart-frontend/src/Components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Categories, Brands } from "../../db";
+import { Slider } from "./Slider";
+// import { useFilterProvider } from "../../Context/FilterContext/FilterProvider";
+
+import "./home.css";
+export const Home = () => {
+  //const { filterDispatch } = useFilterProvider();
```
Comment (saadahmadhere) [inline front-kart-frontend/src/Components/Home/Home.js:9]: if these kind of comments are not of use, they should be removed.

File: front-kart-frontend/src/Components/Home/home.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  display: inline-block;
+}
+
+.category-title {
+  color: var(--dark-secondary-color);
+}
+
+.card-hover {
+  cursor: pointer;
+  box-shadow: 1px 4px 12px 0px var(--light-secondary-color);
```
Comment (saadahmadhere) [inline front-kart-frontend/src/Components/Home/home.css:20]: it's better to use relative units for values > 12px

File: front-kart-frontend/src/db.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import category1 from "./Components/Home/assets/Jeans-image.webp";
```
Comment (saadahmadhere) [inline front-kart-frontend/src/db.js:1]: I think you have already hosted these images now. If so, it's better to remove the local imports.

**Review by saadahmadhere — COMMENTED** [review]
Well written, nicely structured code. Have made suggestions wherever found needful. To just add one more thing, the PR's could be even more smaller.

File: front-kart-frontend/src/ProductData.js [hunk 4]
```diff
@@ -0,0 +1,383 @@
+export const ProductData = [
+  {
+    id: 1,
```
Comment (lannisterrr) [inline front-kart-frontend/src/ProductData.js:3]: better to use libraries like uuid for giving unique Id

**Review by lannisterrr — COMMENTED** [review]
You should use mockbee for this project. Other than points mentioned above , the code seems well structured.

**Pull request merged**